.eslintrc

```diff
@@ -1,5 +1,11 @@
 {
   "extends": ["airbnb", "prettier"],
   "parser": "@babel/eslint-parser",
-  "env": { "jest": true, "node": true, "browser": true, "jasmine": true }
+  "env": { "jest": true, "node": true, "browser": true, "jasmine": true },
+  "rules": {
+    "no-console": [
+      "warn", 
+      { "allow": ["warn", "error", "info", "time", "timeEnd"] }
+    ] 
+  }
```

Comment:
You are adding additional rules for error management, correct?

Reply:
Yes, updating the rules of the eslint